bsonjs/bson/bson-version.h

```diff
@@ -69,7 +69,7 @@
  * BSON version, encoded as a string, useful for printing and
  * concatenation.
  */
-#define BSON_VERSION_S "1.24.2"
+#define BSON_VERSION_S ""
```

Comment:
Is it expected that BSON_VERSION and BSON_VERSION_S are empty?

Reply:
It was an issue with the vendor script, updated.